app/cypress/e2e/0-ndr-core-tests/gp_view_lloyd_george_workflow.cy.js

```diff
@@ -0,0 +1,115 @@
+import viewLloydGeorgePayload from '../../fixtures/requests/GET_LloydGeorgeStitch.json';
+import searchPatientPayload from '../../fixtures/requests/GET_SearchPatient.json';
+
+const baseUrl = Cypress.env('CYPRESS_BASE_URL') ?? 'http://localhost:3000/';
+
+describe('GP View Lloyd George Workflow', () => {
+    beforeEach(() => {
+        // Arrange
+        navigateToLgPage();
+    });
+
+    it.only('allows a GP user to view the Lloyd George document of an active patient', () => {
```

Comment:
any specific reason, only this test is executed. Should we mention as smoke test (since we still use mock data)

Reply:
my mistake - I was meant to take that out. I included it for ease of creating the test. Will amend!